index.html

```diff
@@ -49,32 +50,171 @@ <h5 class="logo">Ecom</h5>
           <li><a href="#">Login/Signup</a></li>
         </sidebar>
       </div>
+      <!----------- hero section ------------>
       <section class="body">
         <div class="content">
-          <h1>Ecom Website</h1>
+          <h1>Winter Store</h1>
           <p>
             Welcome to ecom website get the exclusive products in less prices
           </p>
-          <button class="primary-btn">Get started</button>
+          <button class="primary-btn">Shop Now</button>
+        </div>
+      </section>
+
+      <!----------- our section ------------>
+      <section class="our-story">
+        <div class="title1"><h1>Welcome to our story</h1></div>
+        <div class="des">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
         </div>
+        <div class="brandname"><h3>Winter Store</h3></div>
       </section>
-      <div class="feature-prod">
-        <div class="feature-card">
-          <div class="image-feature">
+
+      <!----------- categories section ------------>
+      <section class="row-container">
+        <div class="varity-card">
+          <div class="element">
             <img
-              src="https://images.unsplash.com/photo-1554227231-54aa5db01c51?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NXx8cHJvZHVjdCUyMHdpdGglMjB3aGl0ZSUyMGJnfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60"
+              src="https://cdn.pixabay.com/photo/2020/10/26/14/50/pumpkin-5687556__340.jpg"
               alt=""
             />
           </div>
-          <div class="feature-prod-content">
-            <h1 class="feature-prod-heading">Name of product</h1>
-            <p class="feature-prod-desc">description of product and all</p>
+          <div class="text">
+            <div class="box">
+              <span>Theme Parties</span>
+              <p>some content here</p>
+            </div>
           </div>
         </div>
-      </div>
+        <div class="varity-card">
+          <div class="element">
+            <img
+              src="https://cdn.pixabay.com/photo/2017/09/06/21/42/model-2723144__340.jpg"
+              alt=""
+            />
+          </div>
+          <div class="text">
+            <div class="box">
+              <span>Customize Items</span>
+              <p>some content here</p>
+            </div>
+          </div>
+        </div>
+      </section>
+
```

Comment:
In these categories section, if I hover over the image it zooms in but as soon as I hover over the text in between the image, it zooms out. If this is intentional it is fine, if not then you can fix that.

Reply:
yes as the hover is applied on the image so it is not working on the text as text div has a width 100% so that area will not work in hover.

Reply:
apply the hover on the card and then apply the styles on image, it will work.

Reply:
okay, i will try this